pages/common/nest.md

```diff
@@ -0,0 +1,24 @@
+# nest
+
+> The Nest CLI is a command-line interface tool that helps you to initialize, develop, and maintain your NestJS applications.
+> More information: <https://docs.nestjs.com/cli/overview>.
+
+- Display information about installed nest version:
+
+`nest info`
+
+- Create a Nestjs project:
+
+`nest new {{project_name}} {{optional_args}}`
```

Comment:
To someone new to the command `{{optional_args}}` isn't very helpful. We should either specify specific examples or leave them out.